settings.gradle

```diff
@@ -7,4 +7,5 @@ include 'situp-plugins:common'
 include 'situp-plugins:apmtracesource'
 include 'situp-plugins:elasticsearch'
 include 'situp-plugins:lmdb-processor-state'
+include 'situp-plugins:oteltraceprocessor'
```

Comment:
Is this required? The raw processor is part of the apmtracesource right?

Reply:
Yes for now, included it incase we want to move the processor, will remove it.